cypress/support/api/data-import.js

```diff
@@ -0,0 +1,130 @@
+/// <reference types="cypress" />
+
+import getRandomPostfix from '../utils/stringTools';
+
+Cypress.Commands.add('createMappingProfileApi', (mappingProfile) => {
+  return cy.okapiRequest({
+    method: 'POST',
+    path: 'data-import-profiles/mappingProfiles',
+    body: {
+      profile: { ...mappingProfile }
+    },
+  });
+});
+
+Cypress.Commands.add('createActionProfileApi', (actionProfile) => {
+  return cy.okapiRequest({
+    method: 'POST',
+    path: 'data-import-profiles/actionProfiles',
+    body: {
+      ...actionProfile
+    },
+  });
+});
+
+Cypress.Commands.add('createJobProfileApi', (jobProfile) => {
+  return cy.okapiRequest({
+    method: 'POST',
+    path: 'data-import-profiles/jobProfiles',
+    body: jobProfile
+  });
+});
+
+Cypress.Commands.add('getId', () => {
+  return cy.okapiRequest({
+    path: 'data-import/uploadDefinitions',
+    searchParams: {
+      query: '(status==("NEW" OR "IN_PROGRESS" OR "LOADED")) sortBy createdDate/sort.descending',
+      limit: 1
+    },
+  });
+});
+
+const addJobProfileRelation = (expectedRelations, actionProfileId) => {
+  const defaultJobProfileRelation = {
+    masterProfileType: 'JOB_PROFILE',
+    detailProfileType: 'ACTION_PROFILE'
+  };
+  const newJobProfileRelation = { ...defaultJobProfileRelation };
+  newJobProfileRelation.order = expectedRelations.length;
+  newJobProfileRelation.detailProfileId = actionProfileId;
+  expectedRelations.push(newJobProfileRelation);
+};
+
+Cypress.Commands.add('addJobProfileRelation', (expectedRelations, actionProfileId) => {
+  const defaultJobProfileRelation = {
+    masterProfileType: 'JOB_PROFILE',
+    detailProfileType: 'ACTION_PROFILE'
+  };
+  const newJobProfileRelation = { ...defaultJobProfileRelation };
+  newJobProfileRelation.order = expectedRelations.length;
+  newJobProfileRelation.detailProfileId = actionProfileId;
+  expectedRelations.push(newJobProfileRelation);
+});
+
+/**
+ * @name cy.createLinkedProfiles
+ * @param {Object} testData - Test data to create and link mapping, action and job profiles.
+ * @param {Object} testData.instanceMappingProfile
+ * @param {Object} testData.instanceActionProfile
+ * @param {Object} testData.holdingsMappingProfile
+ * @param {Object} testData.holdingsActionProfile
+ * @param {Object} testData.itemMappingProfile
+ * @param {Object} testData.itemActionProfile
+ * @param {Object} testData.jobProfileForCreate
```

Comment:
@TetianaParanich Let's change this `jsDoc` since your `testData` is an array of objects now.
```suggestion
 * @param {Object[]} testData - Test data to create and link mapping, action and job profiles.
```